cpmonitor/tests/migrations_test.py

```diff
@@ -0,0 +1,190 @@
+import os
+import json
+from django.apps import apps
+from django.apps.registry import Apps
+from django.db.models import ManyToOneRel
+from django_test_migrations.migrator import Migrator
+
+from cpmonitor.models import Task
+
+app_name = "cpmonitor"
+
+
+def read_fixture(fixture_name: str, app_name: str, migration_state_apps: Apps):
+    """
+    Reads data from a JSON fixture file of the given application into a given applications state.
+
+    This is a drastically simplified version of `django.core.management` "loaddata" command,
+    which cannot be used, since it uses `django.apps.apps` and we need to use `old_state.apps`.
```

Comment:
I don't get why you cannot use loaddata

Reply:
Because `django.apps.apps` uses to the current / latest model version with all migrations applied. The load of older data simply fails.

Reply:
Put more explanation in the docstring.